pom.xml

```diff
@@ -308,6 +307,7 @@
         <module>web-common</module>
         <module>servlet-dist</module>
         <module>servlet-feature-pack</module>
+        <module>feature-pack</module>
```

Comment:
I double-checked, and it seems you're right: my local build is working fine now even without this patch. I'm wondering though if something is now locally cached.

You have a better theory to explain the build error I had ?

  > [ERROR] Failed to execute goal on project wildfly-feature-pack: Could not resolve dependencies for project org.wildfly:wildfly-feature-pack:pom:10.0.0.Alpha6-SNAPSHOT: Failure to find org.wildfly:wildfly-servlet-feature-pack:zip:10.0.0.Alpha6-SNAPSHOT in http://repository.jboss.org/nexus/content/groups/public/ was cached in the local repository, resolution will not be reattempted until the update interval of jboss-public-repository-group has elapsed or updates are forced -> [Help 1]